O/OpenCV/build_tarballs.jl

```diff
@@ -123,8 +123,8 @@ dependencies = [
     Dependency(PackageSpec(name="Qt5Base_jll", uuid="ea2cea3b-5b76-57ae-a6ef-0a8af62496e1"))
     Dependency(PackageSpec(name="Libglvnd_jll", uuid="7e76a0d4-f3c7-5321-8279-8d96eeed0f29"))
     BuildDependency(PackageSpec(name="libjulia_jll"))
-    Dependency(PackageSpec(name="libcxxwrap_julia_jll"))
+    Dependency(PackageSpec(name="libcxxwrap_julia_jll", uuid="3eaa8342-bff7-56a5-9981-c04077f7cee7"); compat="0.11")
 ]
 
 # Build the tarballs, and possibly a `build.jl` as well.
-build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies; julia_compat="1.6", preferred_gcc_version = v"9")
+build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies; julia_compat="1.10", preferred_gcc_version = v"9")
```

Comment:
Any reason why this was changed?

Reply:
Because otherwise you canno use OpenCV with Julia 1.10.x (at least).

Reply:
I'm confused. Can you elaborate on why?

Reply:
Because it generates several errors?

@Moeasy64 (and friends) has the details.

This appears to be a well known issue with OpenCV and Julia 1.10.x (worked fine in 1.9.x) since January.  Alas, it has not been resolved yet.

Cfr. https://github.com/JuliaPackaging/Yggdrasil/pull/7941

Reply:
> Cfr. #7941

That PR didn't change the compat bound. I still don't understand why that should change.

Reply:
> To clarify, I think this was a misunderstanding, the Julia compat bound oof 1.6 means it will work with Julia 1.6 and above, so leaving this at 1.6 is fine.

I think instead that this is the issue.  The current libcxxwrap library (which is required by OpenCV.jl) does _not_ work in Julia 1.10.

Any suggestions about how to make things right?

Reply:
> I think instead that this is the issue. The current libcxxwrap library (which is required by OpenCV.jl) does not work in Julia 1.10.

Changing this line is totally unrelated to that problem and insisting on that will bring us nowhere. I still didn't get a real motivation for why this line should be changed.

Reply:
Agree. This should just be left at 1.6.